forge/tests/error/main.rkt

```diff
@@ -56,7 +56,8 @@
     (list "example_impossible.frg" #rx"impossible")
     (list "failed_theorem.frg" #rx"failed.")
     (list "failed_unsat.frg" #rx"Failed test")
-    (list "failed_sat.frg" #rx"Failed test")
+    (list "failed_sat.frg" #rx"Failed test") 
+    (list "properties_undirected_tree_error.rkt" #rx"failed.")
```

Comment:
Oh ... We really need a better regex here. The error needs to give a hint about what went wrong, beyond `temporary-name1 failed` and giving an instance.

Also, split the `properties_....` file into two, one for each error. This test runner catches only the first error in a file.

Reply:
To fix later